includes/options/class-amp-options-manager.php

```diff
@@ -439,6 +440,44 @@ public static function render_cache_miss_notice() {
 		);
 	}
 
+	/**
+	 * Render PHP-CSS-Parser conflict notice.
+	 *
+	 * @return void
+	 */
+	public static function render_php_css_parser_conflict_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		if ( AMP_Style_Sanitizer::has_required_php_css_parser() ) {
+			return;
+		}
+
+		try {
+			$reflection = new ReflectionClass( 'Sabberworm\CSS\CSSList\CSSList' );
+			$source_dir = str_replace(
+				trailingslashit( WP_CONTENT_DIR ),
+				'',
+				preg_replace( '#/vendor/sabberworm/.+#', '', $reflection->getFileName() )
+			);
+
+			printf(
+				'<div class="notice notice-warning"><p>%s</p></div>',
+				sprintf(
+					/* translators: %s is location where conflicting lib was found */
+					esc_html__( "A conflicting version of PHP-CSS-Parser appears to be installed by another plugin/theme (located in '%s'). Because of this CSS processing will be limited, and tree shaking will not be available.", 'amp' ),
```

Comment:
This notice looks good when *Mailpoet* is active:

<img width="1203" alt="notice-mailpoet-active" src="https://user-images.githubusercontent.com/4063887/49979817-61349700-ff16-11e8-88db-7f7b05e208bd.png">